app/repositories/repository_challenge.rb

```diff
@@ -0,0 +1,69 @@
+module RepositoryChallenge
+
+  def self.collection_aggregate(*args)
+    Challenge.collection.aggregate(args.flatten)
+  end
+
+  def self.count_entries_query
+    collection_aggregate({
```

Comment:
Could you please add a short description / explainer of what this query is doing, plus docs references to mongo syntax?

Reply:
I'll do that

Reply:
done